.github/workflows/ci_dockers.yml

```diff
@@ -17,6 +17,10 @@ on: # Trigger the workflow on push or pull request, but only for the master bran
       - ".github/workflows/events-nightly.yml"
       - "setup.py"
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Can you explain what this does?

Reply:
cancel workflow if in progress

Reply:
And what does that mean? Why would we want that?

Reply:
have you checked the describtion?

Reply:
@Borda the description links a 10 min video which does not explain it. I think what Carlos was looking for was more a quick description like this one: https://github.blog/changelog/2021-04-19-github-actions-limit-workflow-run-or-job-concurrency/

cancel-in-progress is explained [here](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#concurrency) but I don't really understand it well 😅 I think it just means when you push a new commit, it should cancel the current job.